.github/workflows/release.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: docker://antonyurchenko/git-release:v3.5.0
         env:
           GITHUB_TOKEN: {{'${{ secrets.TOOLS_BOT_PAK }}'}}
-          RELEASE_NAME_PREFIX: "s1-gunw "
+          RELEASE_NAME_PREFIX: "s1-enumerator "
```

Comment:
Should check on this action -- it assumes a `CHANGELOG.md`, so you may want to add all that as well.